docs/troubleshooting.md

```diff
@@ -46,4 +47,38 @@ You can temporarily collect information to let the session unification process a
 identified profile when the user signs up. If the user does not signs up, the personalization engine will anonymize 
 the profile after the session has expired.
 
+## I installed a plugin, but it didn’t work
+
+Forgetting to import the plugin into the same file where we initialized the SDK is the most common cause for a plugin not work. The import is essential for the SDK to discover and load all the plugins you want to use. You also need to enable the plugin explicitly.
+
+The following example shows how to enable the Google Analytics Plugin using the default settings:
+
+```js
+import croct from '@croct/plug';
+
+// Install the plugin
+import '@croct/plug-google-analytics';
+croct.plug({
+    plugins: {
+        // Enable the plugin
+        googleAnalytics: true,
+    }
+});
+```
+
+If the problem persists, check if the plugin has been found and loaded. To do this, follow the steps below:
+
+1. Enable the debug mode
+```js
+croct.plug({
+    debug: true,
+    plugins: {
+        googleAnalytics: true,
+    }
+});
+```
+2. Open your browser and check if there is an error like "Plugin ‘pluginName’ is not registered" in the console
```

Comment:
Wrong quotes

Reply:
2. Open your browser and check if there is an error like _"Plugin 'pluginName' is not registered"_ in the console

Reply:
keep in italic, pls